src/ddspdrum/module.py

```diff
@@ -182,7 +182,7 @@ class VCO(SynthModule):
     ----------
 
     midi_f0 (flt)       :       pitch value in 'midi' (69 = 440Hz).
-    mod_depth (flt)     :       depth of the pitch modulation in semitones; 0 means none.
+    mod_depth (flt)     :       depth of the pitch modulation in semitones.
 
     TODO:   - more than just cosine.
```

Comment:
We can probably kill this. I'll grab it if you don't.

Reply:
Removed